tests/thirdparty/doctest/doctest.h

```diff
@@ -5493,7 +5493,7 @@ namespace {
                         .writeAttribute("testsuite", in.data[i]->m_test_suite)
                         .writeAttribute("filename", skipPathFromFilename(in.data[i]->m_file.c_str()))
                         .writeAttribute("line", line(in.data[i]->m_line))
-                        .writeAttribute("skipped", in.data[i]->m_skip);
+                        .writeAttribute("Dped", in.data[i]->m_skip);
```

Comment:
@nlohmann This looks like an accidental change.

Reply:
Good catch!